sympy/solvers/tests/test_ode.py

```diff
@@ -36,6 +36,11 @@ def test_linear_2eq_order1():
     Eq(y(t), -6*sqrt(3)*C1*exp(-6*sqrt(3)*t) + 6*sqrt(3)*C2*exp(6*sqrt(3)*t))]
     assert dsolve(eq1) == sol1
 
+    eq11 = (Eq(diff(C1(t),t), 9*C2(t)), Eq(diff(C2(t),t), 12*C1(t)))
+    sol11 = [Eq(C1(t), 9*C1*exp(-6*sqrt(3)*t) + 9*C2*exp(6*sqrt(3)*t)), \
+    Eq(C2(t), -6*sqrt(3)*C1*exp(-6*sqrt(3)*t) + 6*sqrt(3)*C2*exp(6*sqrt(3)*t))]
+    assert dsolve(eq11) == sol11
+
```

Comment:
Ah, I didn't think of this case!  I was thinking of `C1` being on the RHS like `C1*x(t)`.  Let's definitely keep this.

See `test_dsolve_linsystem_symbol` for what I was thinking, but with `C1` instead of epsilon...  Is that do-able too?  (The positive assumption might be necessary for dsolve to work, which is bad for this test I expact.  Maybe you can put it as `diff(x(t),t) == x(t) + y(t) + C1`...)

Can you move your test to a new standalone test function as well (i.e., `def test_C1_function_9239():`, down at the bottom).